cluster/eks/composition.yaml

```diff
@@ -35,7 +35,6 @@ spec:
                     }
                 ]
               }
-      name: controlplaneRole
```

Comment:
i don't think these base resource name fields should be removed - they were added in #63 as part of the effort to adopt the latest best practices in compositions

this comment applies to other composition files in this PR too

Reply:
awesome @garreeoke - can you give me a ping (@ mention me here) when this PR is ready to take another review pass on? thank you!